src/main/java/knox/spring/data/neo4j/services/DesignSpaceService.java

```diff
@@ -325,7 +326,31 @@ public void mergeBranches(String targetSpaceID, List<String> inputBranchIDs,
         
         saveDesignSpace(targetSpace);
     }
-	
+
+	public void reverseDesignSpaces(String inputSpaceID) {
+		reverseDesignSpaces(inputSpaceID, inputSpaceID);
+	}
+
+	public void reverseDesignSpaces(String inputSpaceID, String outputSpaceID) {
+		List<String> inputSpaceIDs = new ArrayList<>();
+		inputSpaceIDs.add(inputSpaceID);
+		List<NodeSpace> inputSpaces = new ArrayList<NodeSpace>(inputSpaceIDs.size());
+
+		DesignSpace outputSpace = loadIOSpaces(inputSpaceIDs, outputSpaceID, inputSpaces);
+
+		ReverseOperator.apply(inputSpaces.get(0), outputSpace);
```

Comment:
Since Operator is only applied to the first inputSpace, it seems misleading (and incorrect) for the user to be able to put in multiple inputSpaces.